src/redis.c

```diff
@@ -4,10 +4,13 @@ typedef struct Meta {
     redisContext *c;
     redisReply   *reply;
     char         *topic;
+    size_t        pipe_cur;
+    size_t        pipe_max;
+    bool          pipe_full;
 } *Meta;
 
 Meta
-redis_meta_init(char *host, char *topic)
+redis_meta_init(char *host, char *topic, int pipe_max)
```

Comment:
pipe_max should be size_t

Reply:
ack. fixup changes `pipe_max` to `size_t`.
Ideally this also requires changes to the option parsing code (`atoi` error checking & int vs. size_t bounds checking), but that's best fixed on a different issue as other parameters have similar issues.